pkg/objstore/objstore.go

```diff
@@ -26,6 +26,9 @@ type Bucket interface {
 
 	// Delete removes the object with the given name.
 	Delete(ctx context.Context, name string) error
+
+	// GetBucket returns the bucket name for the provider.
+	GetBucket() string
```

Comment:
Can we update this function name ... `Name()`? We tend not to have Getters in Go and although its returning `string` on first glance I would assume it returns the bucket.